src/MyEventEmitter.js

```diff
@@ -1,12 +1,64 @@
-'use strict';
-
 export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this._events = {};
+  }
+
+  on(name, listener) {
```

Comment:
what if i pass a string | object as a `listener`?
or a function | object as `name`?

you would want to validat the listener and name types.

Reply:
I wanted to make it private but eslinl does not accept #property so as a old convention I made it _property